runtime/sema/errors.go

```diff
@@ -53,6 +53,16 @@ func (e *unsupportedOperation) Error() string {
 	)
 }
 
+// InvalidPragmaError
+
+type InvalidPragmaError struct {
+	Message string
+}
+
```

Comment:
As mentioned above, this error needs position information. Maybe just embed `ast.Range` like in the other errors. Also, the error should conform to `SemanticError` (done by just implementing (`isSemanticError()`):


```suggestion
type InvalidPragmaError struct {
	Message string
	ast.Range
}

func (*InvalidPragmaError) isSemanticError() {}
```